extendr-api/src/robj/try_from_robj.rs

```diff
@@ -178,6 +178,30 @@ impl TryFrom<Robj> for Vec<f64> {
     }
 }
 
+impl TryFrom<Robj> for Vec<Bool> {
+    type Error = Error;
+
+    fn try_from(robj: Robj) -> Result<Self> {
+        if let Some(v) = robj.as_logical_slice() {
+            Ok(Vec::from(v))
+        } else {
+            Err(Error::ExpectedInteger(robj))
+        }
+    }
+}
+
+impl TryFrom<Robj> for Vec<u8> {
```

Comment:
Is your assumption here that `Vec<u8>` should only be used when the R object is a raw vector, and never if it is an R numeric or integer? If yes, then I think this should be documented.

Reply:
I've documented all the conversions.